### ISSUE TITLE
'error while loading shared libraries: libswscale.so.4'

### ISSUE THREAD

**Issue by makkus** [MilanSuk/SkyAlt#1]
Hiya. Just tried to run this on the latest Ubuntu (19.10), and getting this error:

```❯ ./skyalt
./skyalt: error while loading shared libraries: libswscale.so.4: cannot open shared object file: No such file or directory
```
I've got both the ``libswscale5`` and ``libswscale-dev`` installed, if that makes any difference:

```
❯ apt search libswscale
Sorting... Done
Full Text Search... Done
libswscale-dev/eoan,now 7:4.1.4-1build2 amd64 [installed]
  FFmpeg library for image scaling and various conversions - development files

libswscale5/eoan,now 7:4.1.4-1build2 amd64 [installed,automatic]
  FFmpeg library for image scaling and various conversions - runtime files
```

**Comment by MilanSuk** [comment]
Hi Markus,
I compile it on Ubuntu 18.04.3 LTS. I run "apt search libswscale" and it has libswscale4.

If you compile with gcc, you can simply compile SkyAlt by your self. It's only 3 commands, you can find them in README file. It should take around 1-2minutes. The output is only one execution file, which you can copy to the folder which you downloaded from the website.

Next version I will compile for both Ubuntu 18 and 19. Or do you know other options how to distribute SkyAlt for Linuxes?

**Comment by MilanSuk** [comment]
Or I could add all libraries to .zip file and distribute SkyAlt with them. What do you think?

**Comment by makkus** [comment]
Yes, I could compile it myself, but figured you might want to know, since your average customer would probably not be able to. Although, they probably wouldn't use the Linux version, so...

Not sure what the best course of action is, depends on how important Linux support is for you. Maybe you could statically compile the whole thing? Or, alternatively provide a snap or flatpak. Or native packages (for that, I can recommend fpm: https://github.com/jordansissel/fpm ).

But ultimately it's up to you, how much effort you want to put into this vs. how much you/your users get out of it. As I said, your main target audience might not use Linux in the first place...

**Comment by MilanSuk** [comment]
Right now, 35 downloads for Windows and 33 downloads for Linux. Personally, I love Linux! I will fix it.
Thanks for tips.
